packages/tooltips/examples/default.md

```diff
@@ -0,0 +1,100 @@
+The `Tooltip` component acts as a high-abstraction API over the
+[useTooltip()](https://www.npmjs.com/package/@zendeskgarden/container-tooltip)
+hook.
+
+All props passed to the root element are proxied onto the tooltip element.
+
+```jsx
+const { Button } = require('@zendeskgarden/react-buttons/src');
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Field, Label, Toggle } = require('@zendeskgarden/react-forms/src');
+const {
+  Dropdown,
+  Select,
+  Field: SelectField,
+  Label: SelectLabel,
+  Menu,
+  Item
+} = require('@zendeskgarden/react-dropdowns/src');
+
+initialState = {
+  placement: 'top',
+  showArrow: true,
+  type: 'dark'
+};
+
+const StyledSpacer = styled.div`
+  margin-bottom: ${props => props.theme.space.sm};
+`;
+
+<Grid>
+  <Row>
+    <Col md={4}>
+      <Well recessed>
+        <StyledSpacer>
+          <Dropdown selectedItem={state.placement} onSelect={placement => setState({ placement })}>
```

Comment:
I noticed that the dropdown loses focus after selecting an option via keyboard. After selecting an option via keyboard, the `activeElement` becomes `<body>`. Is there a way to retain focus on the dropdown after selecting an option?

![focus-lost-1](https://user-images.githubusercontent.com/1811365/68904927-26257f80-06f4-11ea-85a8-9e6b0f8ab7e9.gif)

Reply:
Resolved with answer above.